scripts/microbench/microbench_parser.py

```diff
@@ -1,53 +1,153 @@
+parser.add_argument('-n', '--node', type=str, help='jenkin node lable')
+parser.add_argument('--html_off', action='store_true', help='turn off html report generate')
+args = parser.parse_args()
+
+path = args.log_path if args.log_path else os.getcwd()
+files = os.listdir(path)
+torchbench=""
+huggingface=""
+timm=""
+for file in files:
+    if file.startswith("multi_threads_opbench_torchbench"):
+        torchbench=path+'/'+file
+    elif file.startswith('multi_threads_opbench_huggingface'):
+        huggingface=path+'/'+file
+    elif file.startswith('multi_threads_opbench_timm'):
+        timm=path+'/'+file
+
+def report_generate(file):
+    success_ops = {}
+    error_ops = {}
+    skipped_ops = {}
+    with open(file) as reader:
+        lines = reader.readlines()
+        op = ""
+        for line in lines:
+            if line.strip() == "":
+                continue
+            if line.startswith("Running"):
+                op = line.split(" ")[1].strip()
+            # Skipping aten.convolution_backward.default, no inductor impl
+            elif line.startswith("Skipping"):
+                op = line.split(",")[0].split(" ")[1].strip()
+                reason = line.split(",")[1].strip()
+                if op not in skipped_ops.keys():
+                    skipped_ops[op] = reason
+                    # print(op+", " + reason)
+            # error aten.stack.default
+            elif line.startswith("error"):
+                op = line.split(" ")[1].strip()
+                if op not in error_ops.keys():
+                    error_ops[op] = "error"
+                    # print(op + ", error")
+            if line.startswith("Inductor"):
+                speedups = line.split("[")[-1].split("]")[0]
+                if op not in success_ops.keys():
+                    success_ops[op] = speedups
+                    # print(op+", "+speedups)
+    results = []
+    for op in sorted(success_ops):
+        results.append(op + ", " + success_ops[op]+ "\n")
+    for op in sorted(skipped_ops):
+        results.append(op + ", " + skipped_ops[op]+ "\n")
+    for op in sorted(error_ops):
+        results.append(op + ", " + error_ops[op]+ "\n")
+    
+    r=pd.DataFrame(results)
+    data=pd.DataFrame(r[0].str.split(", ",expand=True))
+    for i in range(4):
+        data[i]= pd.to_numeric(data[i],errors='ignore')
+    data.sort_values(by=[3],inplace=True)  
+    data=data.style.highlight_between(subset=[3], axis=0, left=0, right=1, inclusive='both', props='color:black;background-color:#FFDEAD')
+    return data
+
+
+for file in torchbench,huggingface,timm:
+    if not file.isspace():
+        try:
+            data=report_generate(file)        
+        except:
+            raise Exception('==========error occurred in generating report from '+file+'========')
+
+header=["op_name", "speedup_0.2", "speedup_0.5", "speedup_0.8"]
+h = pd.DataFrame(columns=header)
+with ExcelWriter('op-microbench-'+args.output+'.xlsx',mode="w") as writer:

```

Comment:
Seems the excel file name used in different place, we'd better to define a `report_name`  var for it

Reply:
done